src/components/ProtectedRoute.js

```diff
@@ -46,6 +65,16 @@ class ProtectedRoute extends Component {
             )
           )}
         />
+        <Snackbar
+          open={!!notification.message}
+          anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
```

Comment:
Seems like the best spot among the options. If we want a custom component later to give us more control, that's fine; for now, I like this.